pkg/commands/clusterbuilder/create.go

```diff
@@ -152,6 +152,9 @@ func create(name string, flags CommandFlags, ch *commands.CommandHelper, cs k8s.
 		if err != nil {
 			return err
 		}
+		if err := waiter.Wait(cb); err != nil {
+			return err
+		}
```

Comment:
just an observation... which applies to all the other files calling on `waiter.Wait()`
IF and when `waiter.Wait()` errors, the user is not informed of the resource object that was created.
And not sure, if we care enough to inform of resource creation & of waiting on resource to be ready separately. 

To be fair, I think this applies to `ch.PrintObj(..)` calls too

Reply:
the error will say "<resource> not ready" which I think is informative enough